telemetry/definitions/commonDefinitions.json

```diff
@@ -566,6 +571,11 @@
             "unit": "Milliseconds",
             "metadata": [{ "type": "result" }]
         },
+        {
+            "name": "cloudformation_createProject",
+            "description": "Called when creating a CloudFormation project",
+            "metadata": [{ "type": "templateName" }]
```

Comment:
We should emit result for both metrics, because the process could fail (or be cancelled)

Reply:
ah good catch!